introduction.md

```diff
@@ -0,0 +1,7 @@
+{\rtf1\ansi\ansicpg1252\cocoartf1265\cocoasubrtf210
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.